core/object.cpp

```diff
@@ -1723,7 +1723,7 @@ void Object::_bind_methods() {
 	BIND_VMETHOD(MethodInfo("_set", PropertyInfo(Variant::STRING, "property"), PropertyInfo(Variant::NIL, "value")));
 #ifdef TOOLS_ENABLED
 	MethodInfo miget("_get", PropertyInfo(Variant::STRING, "property"));
-	miget.return_val.name = "var";
+	miget.return_val.name = "Variant";
```

Comment:
I'm not so sure about this line. For GDNative and such it makes sense, but I think GDScript shows this as `var` which is pretty cool. Not everybody who uses GDScript knows about Variant, so it might be confusing for them if there's `Variant` and then **no** documentation about it 😄 

Maybe just checking for `"var"` in the JSON generator and putting in `"Variant"` is better than changing it globally?